Samples/Flix/src/main/java/dev/testify/samples/flix/ui/common/composeables/CreditStrip.kt

```diff
@@ -0,0 +1,72 @@
+package dev.testify.samples.flix.ui.common.composeables
```

Comment:
The `composeables` package name seems redundant

Reply:
I'm open to changing it. I somewhat think this will become more clear once we start adding screenshot tests. How passionate do you feel about it?